packages/lending/src/protocols/radiant-v2/configs.ts

```diff
@@ -62,13 +62,62 @@ export const configs: Config[] = [
         rToken: mainnetTokens.rwstETH,
         used: { deposit: true, withdraw: true, borrow: true, repay: true, flashLoan: true },
       },
+      {
+        asset: mainnetTokens.sDAI,
+        rToken: mainnetTokens.rsDAI,
+        used: { deposit: true, withdraw: true, borrow: true, repay: true, flashLoan: true },
+      },
```

Comment:
We didn't add sDAI initially because the price on Paraswap V5 was very bad at that time. However, I just checked, and it seems Paraswap V5 now supports the routing of sDAI to DAI through mint/burn. Therefore, it's a good time to add sDAI!

Reply:
I also checked this with @LuPoYi and Dan and they agreed to add sDAI back.

Reply:
OK, it seems they weren't aware of the unfavorable price, anyway let's add sDAI.